src/models.py

```diff
@@ -931,6 +959,45 @@ def _lm_forward(self, batch, task, predict):
             pass
         return out
 
+    def _lm_only_lr_forward(self, batch, task, predict):
+        """Only left to right pass for LM model - non-bidirectional models.
+        Args:
+            batch: indexed input data
+            task: (Task obejct)
+            predict: (boolean) predict mode (not supported)
+        return:
+            out: (dict)
+                - 'logits': output layer, dimension: [batchSize * timeSteps * 2, outputDim]
+                            first half: [:batchSize*timeSteps, outputDim] is output layer from forward layer
```

Comment:
Clean up. Why the ':'? Why mixed spacing around *?

Reply:
Cleaned up args.